src/options.ts

```diff
@@ -1,4 +1,5 @@
 import fs from 'fs'
+import process from 'process'
```

Comment:
isn't process a global variable?

Reply:
It is. However the node doc recommends to import it.. I can delete this if you want.